assets/js/googlesitekit/datastore/site/urls.js

```diff
@@ -112,20 +112,33 @@ export const selectors = {
 	 * Gets the Site Kit documentation URL.
 	 *
 	 * @since 1.80.0
+	 * @since n.e.x.t Added the `hasProperty` argument.
 	 *
-	 * @param {Object} state Data store's state.
-	 * @param {string} slug  The slug of the documentation page.
+	 * @param {Object}  state       Data store's state.
+	 * @param {string}  slug        The slug of the documentation page.
+	 * @param {boolean} hasProperty Optional hash property to link to a specific section of the support page.
```

Comment:
I understand the idea here but it won't work as expected since the selector builds a URL to our redirect endpoint on the service. The redirect has already been defined to include the fragment/anchor for the given slug, so no changes should be needed here. It's also worth noting that even if we could add the fragment here, it's better to keep the relationship abstract to allow for changes to the destination URL without requiring changes in the plugin which is one of the primary purposes of the redirect endpoint.